site/en/docs/privacy-sandbox/feedback/index.md

```diff
@@ -228,6 +228,20 @@ groups** column of the tables in the [**Proposals** section](#proposals).
 
 {% Details %} {% DetailsSummary %}
 
+### Chrome-facilated testing
+
+**_We are seeking feedback on the [Chrome-faciliated testing plans](/docs/privacy-sandbox/chrome-testing/) along with tracking issues from initial third-party cookie deprecation._**
+
+{% endDetailsSummary %}
+
+If you rely on third-party cookie data for site functionality, we've launched a [public issue tracker](https://goo.gle/report-3pc-broken), where you can report site issues resulting from third-party cookie deprecation.
+
+Additionally, Chrome will deprecate 1% of third party cookies in Q1 2024, and we'll work closely with the CMA before taking further steps to expand deprecation. You can provide [feedback on GitHub](https://github.com/GoogleChromeLabs/privacy-sandbox-dev-support/labels/chrome-testing) as to the appropriate fraction of traffic to devote to this subset of [Chrome-facilitated testing](/docs/privacy-sandbox/chrome-testing/).
```

Comment:
what are they providing feedback about?

Reply:
Could be worded better, but this is one that went through a few rounds of feedback, so I'll just leave as is.